### ISSUE TITLE
Need updated pxegrub file to address OS-3634 and OS-3635

### ISSUE THREAD

**Issue by nshalman** [joyent/sdc-booter#1]
OS-3634 and OS-3635 improve how grub deals with serial ports.
Someone on IRC ran into a problem that was addressed by replacing the current tftpboot/pxegrub file with a fresh one from the proto dir of a platform build.

Quick fix is to grab a fresh one and check it in.
A more thorough though more complicated fix is to package up pxegrub in the boot tarball generated by the build process (tweak a manifest in illumos-joyent e.g. https://github.com/nshalman/illumos-joyent/commit/c77288706a252fab66be2d556206cd78430a3588) and then have sdc-booter somehow grab pxegrub out of that as a build step.

**Comment by trentm** [comment]
@rgulewich ping, thoughts?

**Comment by sperreault** [comment]
Could we just use a quick fix here and copy a pxegrub file that works (i'm doing this manually right now)

**Comment by nshalman** [comment]
I agree. One of these two tickets (this one and #2) should be fixed by checking in a working binary. The other could be left open for the possibility of the fancier fix.

**Comment by nshalman** [comment]
RELENG-608 (https://github.com/joyent/illumos-joyent/commit/62890e2a24ac142b8b3f5bbbdee8c0ed9ee3ba49) provides the raw materials for both the quick fix and the better fix.

**Comment by nshalman** [comment]
@jclulow I just noticed https://smartos.org/bugview/HEAD-2315

While you're doing GRUB related stuff, perhaps you could take a look at this issue.
